src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -381,6 +389,22 @@ void Instance::HandleAddOrUpdateThreadNetwork(HandlerContext & ctx, const Comman
         mpDriver.Get<ThreadDriver *>()->AddOrUpdateNetwork(req.operationalDataset, debugText, outNetworkIndex);
     FillDebugTextAndNetworkIndex(response, debugText, outNetworkIndex);
     ctx.mCommandHandler.AddResponse(ctx.mRequestPath, response);
+    if (response.networkingStatus == NetworkCommissioningStatus::kSuccess)
+    {
+        KickBreadcrumbChange(req.breadcrumb);
+    }
+}
+
+void Instance::KickBreadcrumbChange(const Optional<uint64_t> & breadcrumb)
```

Comment:
Should probably be called `UpdateBreadcrumb`.

Reply:
Updated